src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListService.cs

```diff
@@ -163,38 +163,30 @@ private void EnsureHashExists(FileData file)
             var hasSha256Hash = file.Hashes.Any(x => x.Algorithm == AlgorithmKind.Sha256);
             if (!hasSha256Hash)
             {
-                string content = null;
-
-                if (!string.IsNullOrWhiteSpace(file.Contents.Binary))
+                byte[] data = null;
+                if (file?.Contents.Binary != null)
```

Comment:
I did this wrong in the sample I sent you. It needs to be `file.Contents?.Binary` and `file.Contents?.Text`.

Reply:
Fixed